statistics.go

```diff
@@ -13,13 +13,14 @@ import (
 
 // Stats contains a bunch of Stats counters
 type Stats struct {
-	Files       uint64 `json:"files"`
-	Dirs        uint64 `json:"dirs"`
-	SymLinks    uint64 `json:"symlinks"`
-	Size        uint64 `json:"size"`
-	StorageSize uint64 `json:"stored_size"`
-	Transferred uint64 `json:"transferred"`
-	Errors      uint64 `json:"errors"`
+	Files               uint64 `json:"files"`
+	Dirs                uint64 `json:"dirs"`
+	SymLinks            uint64 `json:"symlinks"`
+	Size                uint64 `json:"size"`
+	StorageSize         uint64 `json:"stored_size"`
+	Transferred         uint64 `json:"transferred"`
+	Errors              uint64 `json:"errors"`
+	NonIncludedSymLinks uint64 `json:"non_included_symlinks"`
```

Comment:
I think we should probably simply call this `Excluded` (and then also count all the other excluded items).

Reply:
Good idea!